### DISCUSSION TITLE
Easier way to retrieve the native title

### DISCUSSION THREAD

**Discussion by RedX2501** [webdriverio-community/wdio-electron-service#759] [Q&A] [answered]
```ts
var bw = new BrowserWindow();
bw.setTitle("MainWindow");
```

will set the native window title ([doc](https://www.electronjs.org/docs/latest/api/browser-window#winsettitletitle));

to retrieve this value is there an easier way than:

```ts
await browser.electron.execute((electron) => electron.BrowserWindow.getFocusedWindow()?.getTitle())
```

**Comment by christian-bromann** [comment, marked as answer]
> to retrieve this value is there an easier way than:

I don't think so, how would you suggest this value should be retrieved?

**Reply by RedX2501** [reply]
I was hoping there could be a helper on the `browser.electron` but mostly I was just curious if that was the currently shortest way.
